### PR TITLE
Runtime fields: rename script classes

### PR DESCRIPTION
**Pull request by javanna**
With this commit we rename the script classes used for each mapped field type used for runtime fields. The new naming is a shorter version of the previous one: from e.g. BooleanScriptFieldScrip to BooleanScript . We also move such classes to the existing mapper package.

### CODE REVIEW

**Review by nik9000 — COMMENTED** [review]
I think maybe this is too short. What about `AbstractFieldScript` and `BooleanFieldScript`? Without some word like `Field` in there it makes me think they are "for everything".

**Comment by javanna** [comment]
> I think maybe this is too short. What about AbstractFieldScript and BooleanFieldScript? Without some word like Field in there it makes me think they are "for everything".

I am not sure, they are still in the runtimefields package under the runtime-fields xpack plugin.

**Comment by nik9000** [comment]
> I am not sure, they are still in the runtimefields package under the runtime-fields xpack plugin.

Yeah! Its just that folks don't typically use the package name when they look for stuff. I know I don't. I'm ok either way I guess.

While you are renaming these, would you like to move them? I added them at the root not realizing just how large the plugin would get. Maybe a `script` package?

**Comment by javanna** [comment]
I don't have a strong opinion, I can keep the Field part in the name, it does not hurt.

> While you are renaming these, would you like to move them? I added them at the root not realizing just how large the plugin would get. Maybe a script package?

yea I wanted to mention moving them as well. would they fit in the mapper package, or do we want to go with a script package?

File: x-pack/plugin/runtime-fields/src/test/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptFieldTestCase.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import java.io.IOException;
 import java.util.Map;
 
-public abstract class ScriptFieldScriptTestCase<T> extends ESTestCase {
+public abstract class ScriptFieldTestCase<T> extends ESTestCase {
```
Comment (javanna) [inline x-pack/plugin/runtime-fields/src/test/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptFieldTestCase.java:18]: should this rather be FieldScriptTestCase?

Comment (nik9000) [reply x-pack/plugin/runtime-fields/src/test/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptFieldTestCase.java:18]: That sounds better! Please rename it too!

**Comment by javanna** [comment]
@nik9000 after removing needless renames 3 or 4 times I think this is ready :)

**Review by nik9000 — APPROVED** [review]
LGTM

**Pull request merged**